browser/brave_vpn/win/brave_vpn_wireguard_service/status_tray/status_tray_runner.cc

```diff
@@ -116,8 +116,10 @@ bool StatusTrayRunner::IsVPNConnected() const {
 void StatusTrayRunner::ConnectVPN() {
   if (IsWireguardActive()) {
     wireguard::EnableBraveVpnWireguardService(
-        "", base::BindOnce(&StatusTrayRunner::OnConnected,
-                           weak_factory_.GetWeakPtr()));
+        // passing empty params will reconnect using last known good config.
+        "", "", "", "",
```

Comment:
how did this work before given that the config file was deleted?

Reply:
I was wrong about the config being deleted ☹️ I was looking in the wrong directory. Since the service runs as `LOCALSYSTEM`, it's storing at `C:\Windows\Temp\BraveVpn`